specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.3-preview/examples/UpdateKeyRotationPolicy-example.json

```diff
@@ -0,0 +1,61 @@
+{
+  "parameters": {
+    "vaultBaseUrl": "https://myvault.vault.azure.net/",
+    "key-name": "updateKey01",
+    "api-version": "7.3-preview",
+    "keyRotationPolicy": {
+      "lifetimeActions": [
+        {
+          "trigger": {
+            "timeAfterCreation": "P90D"
+          },
+          "action": {
+            "type": "Rotate"
+          }
+        },
+        {
+          "trigger": {
+            "timeBeforeExpiry": "P30D"
+          },
+          "action": {
+            "type": "Notify"
+          }
+        }
+      ],
+      "attributes": {
+        "rotationEnabled": true,
+        "rotationTime": "P2Y"
+      }
```

Comment:
Consider removing the `rotationEnabled`. For MVP, rotation is enabled when there is a life time action of type `Rotate`, and disabled when there is no such item.

Reply:
I forgot to update it in the example file. It is done now.